### DISCUSSION TITLE
Why can I access a rodauth protected route with an unverified account?

### DISCUSSION THREAD

**Discussion by pboling** [janko/rodauth-rails#262] [Q&A] [answered]
I added this to my controller:
```
  before_action -> { rodauth.require_authentication }
```
I am logged in as a new account that hasn't been verified yet.  Verification is working, but I haven't done it for this user.  I am able to access the route protected by the above before action.  Is this expected?

**Comment by janko** [comment, marked as answer]
Yes, if you're logged in, then you're able to access protected routes. You can either set `create_account_autologin?` to `false` to prevent login, or not enable the `verify_account_grace_period` feature.
